content/client-server-api/modules/instant_messaging.md

```diff
@@ -385,6 +386,63 @@ An example of a media message with a caption is:
 Clients MUST render the caption alongside the media and SHOULD prefer its
 formatted representation.
 
+##### Mathematical messages
+
+{{% added-in v="1.11" %}}
+
+Users might want to send mathematical notations in their messages.
+
+To send mathematical notations clients MUST use the `formatted_body` and
+therefore the `org.matrix.custom.html` format, described above. This makes
+mathematical notations valid on any `msgtype` which can support this format
+appropriately.
+
+Mathematical notations themselves use the `span` or `div` tags, depending
+whether the notation should be presented inline or not. The mathematical
+notation is written in [LaTeX](https://www.latex-project.org/) format using the
+`data-mx-maths` attribute.
+
+The contents of the tag should be a fallback representation for clients that
+cannot render the LaTeX format. The fallback representation could be, for
+example, an image, or an HTML approximation, or the raw LaTeX source. When using
+an image as a fallback, the sending client should be aware of issues that may
+arise from the receiving client using a different background colour. The `body`
+should include a textual representation of the notation.
+
+An example of a mathematical notation is:
+
+```json
+{
+  "msgtype": "m.text",
+  "format": "org.matrix.custom.html",
+  "body": "This is an equation: sin(x)=a/b.",
+  "formatted_body": "This is an equation:
+      <span data-mx-maths=\"\\sin(x)=\\frac{a}{b}\">
+        sin(<i>x</i>)=<sup><i>a</i></sup>/<sub><i>b</i></sub>
+      </span>"
+}
+```
+
+The LaTeX format is poorly defined and has several extensions, so if a client
+encounters syntax that it cannot render, it SHOULD present the fallback
+representation instead. Clients SHOULD, however, aim to support, at minimum, the
+basic LaTeX2e maths commands and the TeX maths commands, with the possible
```

Comment:
Should LaTeX2e and TeX link off to other specifications?

Reply:
I am not sure where we could link for TeX, the [TUG](https://www.tug.org) website? There doesn't seem to be a specification.

Reply:
https://www.latex-project.org/ seems to be the official site for LaTeX, so you can link to that for LaTeX2e.  Probably the closest to thing to an official spec for TeX is Knuth's The TeXbook, but that's only available as a physical book, or as [the `.tex` sources](https://ctan.org/pkg/texbook), which probably isn't the easiest to read. I think that either TUG or the `.tex` source for The TeXbook would be the best thing to link to.

Reply:
Links added